src/items/class-features/perceive_dreamspace_(su).json

```diff
@@ -0,0 +1,115 @@
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.udtZxolGt5mMq9SM]{Dreamer}</p><p class=\"MsoNormal\">You can experience the dreams of sapient beings by entering a trance state and focusing on a particular dreamer or dream motif. You perceive this illusory dreamspace as a physical realm unique to each dreamer with whom you interact and explore with any senses you possess, though you can’t directly communicate with or affect the dreamer. Entering dreamspace is a full action. Once per day while inside dreamspace, you can spend 10 minutes concentrating on a single dreamer with whom you’re already familiar or a specific dream motif.</p><p class=\"MsoNormal\">If you observe a dreamer in dreamspace, the target must succeed at a Will save with a DC equal to 10 + your class level to resist, causing you to learn nothing useful. Otherwise, you learn 1d4 pieces of useful information about the dreamer at the GM’s discretion.</p><p class=\"MsoNormal\">If you observe a dream motif, you return to the waking world with the ability to create a physical manifestation of the motif as a standard action. This can mimic the effects of a 1st-level spell from any spell list. You must manifest the dream motif within 24 hours or until you next enter dreamspace (whichever comes first), or it’s lost.</p><p class=\"MsoNormal\">While inside dreamspace, you can’t perceive or interact with the physical world. Returning your consciousness fully to your physical body takes 1 minute.</p>"
+    },
+    "descriptors": {
+      "acid": false,
+      "air": false,
+      "calling": false,
+      "chaotic": false,
+      "charm": false,
+      "cold": false,
+      "compulsion": false,
+      "creation": false,
+      "curse": false,
+      "darkness": false,
+      "death": false,
+      "disease": false,
+      "earth": false,
+      "electricity": false,
+      "emotion": false,
+      "evil": false,
+      "fear": false,
+      "fire": false,
+      "force": false,
+      "good": false,
+      "healing": false,
+      "language-dependent": false,
+      "lawful": false,
+      "light": false,
+      "mind-affecting": false,
+      "pain": false,
+      "poison": false,
+      "polymorph": false,
+      "radiation": false,
+      "scrying": false,
+      "sense-dependent": false,
+      "shadow": false,
+      "sonic": false,
+      "summoning": false,
+      "teleportation": false,
+      "water": false
+    },
+    "duration": {
+      "units": "",
+      "value": null
+    },
+    "formula": "",
+    "isActive": null,
+    "modifiers": [],
+    "range": {
+      "additional": "",
+      "per": "",
+      "units": "",
+      "value": null
+    },
+    "recharge": {
+      "charged": false,
+      "value": null
+    },
+    "requirements": "1st Level (Mystic)",
+    "rollNotes": "",
+    "save": {
+      "type": "",
+      "dc": null,
```

Comment:
Include the DC

Reply:
At the very bottom of the details tab, there's a saving throw and skill check field. Put the field there. May require a saving throw/skill check activation type. That is the preferred way to do saving throws, not inline formulas showing DCs

Reply:
You've still got the saves in the wrong field. Check other class features with saving throws for examples

Reply:
I think I got it correct now

Reply:
Take out the inline formulas for the DCs as well

Reply:
Fixed